src/modules/launcher/Plugins/Microsoft.Plugin.Program/Microsoft.Plugin.Program.csproj

```diff
@@ -108,10 +93,37 @@
     <PackageReference Include="System.Runtime" Version="4.3.1" />
   </ItemGroup>
   
-<ItemGroup>
```

Comment:
@niels9001 This item group is required to make the internals of this project visible to unit test project. UWP tests are failing for this reason.

Reply:
ok, so TLDR: line 111 somehow got deleted but needs to be there?  correct?

Reply:
@crutkas The following item group needs to be added back. It was deleted while resolving merge conflict :
```
<ItemGroup>
    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
       <_Parameter1>Microsoft.Plugin.Program.UnitTests</_Parameter1>
    </AssemblyAttribute>
</ItemGroup>
```